src/main/java/edu/harvard/iq/dataverse/harvest/server/xoai/DataverseXoaiItem.java

```diff
@@ -51,19 +48,28 @@ public Dataset getDataset() {
         return dataset;
     }
     
-    public Xitem withDataset(Dataset dataset) {
+    public DataverseXoaiItem withDataset(Dataset dataset) {
         this.dataset = dataset; 
         return this; 
     }
     
+    private List<About> about;
+    
     @Override
     public List<About> getAbout() {
-        return null;
+        return about;
```

Comment:
**nitpick:** Shouldn't this return an unmodifiable list to avoid changing the objects? (e. g. via `Collections.unmodifiableList()`)

Reply:
Sure. Or I could just delete these method and rely on the empty list implementation in Item.java; since we have no use for it yet.

Reply:
Making the returned list unmodifiable will in fact make it impossible to add any Abouts to the item; since there are no withAbout() or setAbout() methods yet. Yes, I feel like just removing both the variable and the method from the class, until we have some need/use case for this "about" thing.